src/2_RTB/NormeDiProgetto/content.typ

```diff
@@ -17,46 +17,91 @@ Il presente documento ha come scopo la definizione delle #glossary[best practice
-- *y* si incrementa una volta effettuata una modifica e la sua conseguente revisione;
-- *x* si incrementa quando si effettua la modifica definitiva in vista di una revisione di avanzamento, questo comporta l'azzeramento di *y*.
+=== Issue Tracking System <its>
+Come ITS si utilizza _Github_, attraverso le funzioni di _Project_, _Issues_ e _Pull requests_, che garantisce una struttura all'organizzazione di progetto.
 
-Due modifiche, fatte in momenti diversi, differiscono l'una dall'altra solo se hanno scopi diversi. Ad esempio non è necessario incrementare la versione se viene fatta una modifica alla stessa sezione in due giorni differenti; anche se faccio una modifica, ed essa non viene approvata, non è necessario incrementare la versione con le nuove modifiche proposte dal revisore, dal momento che modifica e revisione "viaggiano" parallelamente.
+Le #glossary[Corsie di Stato] descrivono lo stato attuale delle attività, all'interno del _Project_ nell'_ITS_ sono presenti:
+/ Backlog: attività individuate da svolgere.
+/ Ready: attività individuate per il completamento durante il prossimo #glossary[sprint].
+/ In Progress: attività che sono in corso d'opera da parte dei redattori.
+/ Ready to Review: attività svolte che sono pronte per essere verificate.
+/ In Review: attività in corso di verifica da parte dei verificatori.
+/ Done: attività le cui modifiche sono state verificate e accettate.
+
+=== Strumento di Condivisione
+Per la condivisione veloce o la creazione di bozze si utilizza _Google Drive_. Uno dei suoi principali casi d'uso comprende la collaborazione in tempo reale a grandi parti di testo che poi andranno inserite all'interno della documentazione (ad esempio durante la creazione di un nuovo documento). Viene inoltre utilizzato come sistema per l'immagazzinamento di conoscenze acquisite durante il corso dello svolgimento del progetto.
+
+=== Tracciamento del Tempo Speso
+Al fine di tracciare il tempo speso nel corso del progetto, nei diversi ruoli, si userà uno spreadsheet appositamente creato, disponibile all'interno di _Google Drive_ dove, a fine giornata, ogni membro del team, individualmente, andrà ad inserire le proprie ore *produttive* svolte quel giorno, secondo la sua miglior stima del rapporto tra ore di orologio e ore produttive. Si inserisce _una sola_ riga per ogni giornata e nella descrizione si andranno ad inserire dei brevi titoli rappresentativi delle attività svolte.
+
+== Verifica
+
+=== Elementi Interni al Repository
+La verifica del documento avviene tramite apposito metodo nell'ITS, attraverso la _Pull request_, indicando i punti in cui si richiede la modifica, il motivo della richiesta e una proposta se necessario. Anche in questo caso i verificatori si occupano di spostare la issue di riferimento, nelle corsie appropriate, chiudendole se la verifica è terminata con successo, o spostandola nuovamente nella corsia _In Progress_ quando vengono richieste altre modifiche.
```

Comment:
nella corsia appropriata* e chiudendola*, è una sola issue alla volta. 
l'ultime 2 parole le riscriverei così: "in caso vengano "